spec/controllers/rents_controller_spec.rb

```diff
@@ -49,10 +49,17 @@
         # 201: The request has been fulfilled and has resulted
         # in one or more new resources being created.
         r = create(:rent)
-        post :create, params: { rent: { user_id: r.user, book_id: r.book, from: r.from, to: r.to } } # rubocop:disable Metrics/LineLength
+        post :create, params: { rent: { book_id: r.book, from: r.from, to: r.to } }
         expect(response).to have_http_status(:created)
       end
 
+      it 'We have 4 jobs pendings' do
+        r = create(:rent)
+        post :create, params: { rent: { book_id: r.book, from: r.from, to: r.to } }
+        expect(Sidekiq::Worker.jobs.size).to eq(2 + 2)
+        # 2 jobs for this test and 2 jobs for previous test
```

Comment:
You can clean up sidekiq queue before each example! Also, you can make expectations on which Worker is enqueued to run, check [this doc](https://github.com/mperham/sidekiq/wiki/Testing#testing-worker-queueing-fake).